circle/circle.go

```diff
@@ -20,4 +21,10 @@ func main() {
 	// ++ Your code here! ++
 	//
 	// Hint: Use math.Pi constant (https://pkg.go.dev/math#pkg-constants) to obtain π value.
-}
\ No newline at end of file
+	var r float64
+	const π = math.Pi
```

Comment:
Redefining `math.Pi` as `π` looks pretty and is technically correct (Go allows to use unicode literals).

However,
1) redefining constant doesn't improve readability, and, therefore, should be generally avoided.
2) use only latin alphabet letters and digits while defining literal names (e.g. variable names) (see https://softwareengineering.stackexchange.com/q/16010).

On a side note, if you are interested in languages that heavily rely on non-latin unicode letters, check [APL](https://aplwiki.com/wiki/Simple_examples). This is a great idea for the lightning talk, by the way.